spec/api/v1/requests/users_request_spec.rb

```diff
@@ -243,4 +243,166 @@
       }.to raise_error(ActiveRecord::RecordNotFound)
     end
   end
+
+  describe 'update' do
+    it 'updates a user' do
+      user_details = {
+        email: 'ex@ample.com',
+        date_of_birth: '2001/02/03',
+        name: 'Jane Doe',
+        password: 'password'
+      }
+      headers = {"CONTENT_TYPE"  => 'application/json'}
+      post '/api/v1/users', headers: headers, params: JSON.generate(user_details)
+      created_user = User.last
+      expect(response.status).to eq 201
+      expect(created_user.email).to eq 'ex@ample.com'
+      expect(created_user.date_of_birth.to_s).to eq '2001-02-03'
+      expect(created_user.name).to eq 'Jane Doe'
+      expect(created_user.obituary).to eq nil
+      expect(created_user.password).to eq nil
+      expect(created_user.password_digest).is_a? String
+
+      updated_user_details = {
+        obituary: 'tedious and brief'
+      }
+      headers = {"CONTENT_TYPE"  => 'application/json'}
+      patch "/api/v1/users/#{created_user.id}", headers: headers, params: JSON.generate(updated_user_details)
+
+      expect(response.status).to eq 200
+      updated_user = User.find(created_user.id)
+      expect(updated_user.email).to eq 'ex@ample.com'
+      expect(updated_user.date_of_birth.to_s).to eq '2001-02-03'
+      expect(updated_user.name).to eq 'Jane Doe'
+      expect(updated_user.obituary).to eq 'tedious and brief'
+    end
+
+    it 'can update an obituary to nil' do
```

Comment:
Currently, we have a feature on the front end preventing the user from submitting an empty obituary...I suppose on creation they could start out with none. Do you think we should allow them to submit an empty obituary? We could also have a "delete" button which submits an empty obit

Reply:
Whichever makes the most sense for your workflow. The user object will create with an empty obituary purely as a convenience to front-end. Totally fine whether you want to require users to write the obit or wait until another page...I think the priority for the decision should just be user convenience